packages/shortest/src/ai/prompts/index.ts

```diff
@@ -11,9 +11,10 @@ Expect: 1. Test case to be generated within at least 20 seconds [HAS_CALLBACK]
 IMPORTANT GLOBAL RULES:
 
 1. **Waiting for Conditions**:
-   - Some steps will require waiting before proceeding to the next action.
-   - This waiting can be based on a time delay (e.g., seconds or minutes) or waiting for an element to become visible or clickable.
-   - If the specified condition is not met after the allotted time, the test should be considered failed.
+   - When you are specifically asked to wait for a condition, use the "sleep" tool. Otherwise, we internally handle proper waiting for elements to load.
+   - For time-based waits:
+     - Use { action: "sleep", duration: X } where X is milliseconds
+   - Always wait for the tool to finish before proceeding to the next action. You will recieve a message to continue with your next action once the wait is over. Then validate the condition is met.
```

Comment:
Improving the prompt for the "sleep" tool to instruct AI to use this tool whenever wait is needed to fulfill a condition.